Library/Homebrew/test/utils/git_spec.rb

```diff
@@ -0,0 +1,175 @@
+require "utils/git"
+
+describe Git do
+  before(:all) do
+    git = HOMEBREW_SHIMS_PATH/"scm/git"
+
+    @file = "lib/blah.rb"
+    @repo = Pathname.new("repo")
```

Comment:
Could put this in one of these dirs to avoid the manual cleanup (and be consistent): https://github.com/Homebrew/brew/blob/2c43e069c25e3c7cc255f5cd105df3597246b952/Library/Homebrew/test/spec_helper.rb#L110-L129